state/processor.go

```diff
@@ -31,6 +31,22 @@ type Transaction struct {
 	hash *common.Hash
 }
 
+func NewTransaction(nonce uint64,
+	origin common.Address,
+	destination common.Address,
+	amount *big.Int) *Transaction{
+	return &Transaction{
+		AccountNonce: nonce,
+		Origin: origin,
+		Recipient:&destination,
+		Amount: amount,
+		GasLimit:10,
+		Price:big.NewInt(1),
```

Comment:
even if we don't know what gaslimit and price are right now, maybe it would be nicer to pass them to this function

Reply:
ok